docs/sources/mimir/set-up/jsonnet/configure-autoscaling.md

```diff
@@ -45,8 +45,11 @@ KEDA, as we use it, never changes the number of replicas of Mimir Deployments or
 
 However, if KEDA is not running successfully, there are consequences for Mimir autoscaling too:
 
-- `keda-operator` is down (not critical): changes to `ScaledObject` CRD will not be reflected to the HPA until the operator will get back online. HPA functionality is not affected.
+- `keda-operator` is down (critical): as the operator is the single point of truth, it won'r request the metrics when `keda-operator-metrics-apiserver` requests them and changes to `ScaledObject` CRD will not be reflected to the HPA until the operator will get back online. The deployment (e.g. queriers) will keep working but, in case of any surge of traffic, HPA will not be able to detect it (because of a lack of metrics) and so will not scale up.
 - `keda-operator-metrics-apiserver` is down (critical): HPA is not able to fetch updated metrics and it will stop scaling the deployment until metrics will be back. The deployment (e.g. queriers) will keep working but, in case of any surge of traffic, HPA will not be able to detect it (because of a lack of metrics) and so will not scale up.
```

Comment:
```suggestion
- `keda-operator-metrics-apiserver` is down (critical): HPA is not able to fetch updated metrics and it will stop scaling the deployment until metrics will be back. The deployment (e.g. queriers) will keep working but, if there is a surge of traffic, HPA will not be able to detect it due to a lack of metrics and so will not scale up.
```